### PR TITLE
Update maintainers of sig-docs Portuguese team.

### PR DESCRIPTION
**Pull request by jailton**
Sync maintainers with OWNERS_ALIASES(https://github.com/kubernetes/website/blob/master/OWNERS_ALIASES):
- @devlware 
- @jhonmike 

Add new maintainers to Portuguese team:
- @jailton 
- @rikatz 
- @yagonobre 

/cc @femrtnz @jcjesus

### CODE REVIEW

**Comment by femrtnz** [comment]
+1

**Comment by sftim** [comment]
Hi, with a couple of +1 comments from Portuguese localization approvers I'll be happy to add my endorsement.

In terms of “is the YAML OK”:
/lgtm

**Comment by devlware** [comment]
+1

File: config/kubernetes/sig-docs/teams.yaml [hunk 1]
```diff
@@ -231,13 +231,19 @@ teams:
     description: Approvers for Portuguese content
     members:
     - femrtnz
+    - jailton
     - jcjesus
+    - rikatz
+    - yagonobre
```
Comment (jhonmike) [inline config/kubernetes/sig-docs/teams.yaml:237]: @jailton posso aproveitar o PR eu adicionar o meu user ai? não sei porque não estava :( 

```suggestion
    - yagonobre
    - jhonmike
```

File: config/kubernetes/sig-docs/teams.yaml [hunk 2]
```diff
@@ -231,13 +231,19 @@ teams:
...
     privacy: closed
   sig-docs-pt-reviews:
     description: PR reviews for Portuguese content
     members:
     - femrtnz
+    - jailton
     - jcjesus
+    - rikatz
+    - yagonobre
```
Comment (jhonmike) [inline config/kubernetes/sig-docs/teams.yaml:246]:
```suggestion
    - yagonobre
    - jhonmike
```

File: config/kubernetes/sig-docs/teams.yaml [hunk 3]
```diff
@@ -316,6 +322,7 @@ teams:
     - ianychoi # L10n: Korean
     - inductor # L10n: Japanese
     - irvifa # L10n: Indonesian
+    - jailton # L10n: Portuguese
```
Comment (jhonmike) [inline config/kubernetes/sig-docs/teams.yaml:325]:
```suggestion
    - jhonmike # L10n: Portuguese
```

**Review by jhonmike — APPROVED** [review]
/lgtm ❤️

**Comment by jailton** [comment]
I updated this PR. @devlware  and. @jhonmike are already listed as owners of sig-docs-pt in OWNERS_ALIASES (https://github.com/kubernetes/website/blob/master/OWNERS_ALIASES).

**Comment by sftim** [comment]
Thanks folks
/approve

**Pull request merged**